src/assets/icons/assets/bdeu.svg

```diff
@@ -0,0 +1,12 @@
+<svg width="36" height="36" viewBox="0 0 36 36" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
XUSD and ETHs icons are also missing
![image](https://user-images.githubusercontent.com/44050297/161010199-e5ed370e-e1e9-4320-9eed-9b68b216d4ea.png)

Reply:
That's weird, they show up for me. Did you try rebuilding wallet?
The svg files though have content a bit different from other svgs, which already have been there. I may investigate that if this keeps occuring in the 2nd round of testing.

Reply:
it's working now 👍